crossbeam-utils/src/cache_padded.rs

```diff
@@ -49,7 +52,14 @@ use core::ops::{Deref, DerefMut};
 /// }
 /// ```
 #[derive(Clone, Copy, Default, Hash, PartialEq, Eq)]
-#[repr(align(64))]
+// Starting from Intel's Sandy Bridge, spatial prefetcher is now pulling pairs of 64-byte cache
```

Comment:
Is this also the case for AMD?

Reply:
Judging by a quick skim through AMD's optimization manual, I'd say no.
https://developer.amd.com/wordpress/media/2013/12/55723_SOG_Fam_17h_Processors_3.00.pdf

Reply:
@Mart-Bogdan I would accept a PR to adjust the placement of CachePadded based on the CPU when the user specifies -C target-cpu in rustflag.